pkg/operator/ceph/cluster/mgr/mgr.go

```diff
@@ -301,8 +301,13 @@ func (c *Cluster) updateServiceSelectors(activeDaemon string) error {
 			logger.Infof("no need to update service %q", service.Name)
 			continue
 		}
+		if service.Spec.Selector == nil {
```

Comment:
Should this `if` block be moved above the first `if` block `if service.Spec.Selector[controller.DaemonIDLabel] == activeDaemon` of this `for` loop ?

Reply:
It didn't technically need to be sooner since the comparison of the value just evaluated empty in that case, but for readability I agree it's just simpler to move it first.